HW02_A_Graph-Fails.Rmd

```diff
@@ -1,6 +1,6 @@
 ---
 title: "What went wrong?"
-author: "Robert Gruener"
+author: "Lucas Dyer"

```

Comment:
It's a good practice to keep a record of the author, which I did not notice in my HW.